api/src/main/java/jakarta/data/page/PageRequest.java

```diff
@@ -37,17 +37,27 @@
  * {@code List}, {@code Stream}, {@link Page}, or {@link CursoredPage}.
  * The parameter of type {@code PageRequest} must occur after the method
  * parameters representing regular parameters of the query itself. For
- * example,</p>
+ * example:</p>
  *
  * <pre>
+ * &#64;Find
```

Comment:
Parameter-based automatic query methods don't currently have the ability to do a Between comparison. We will need to wait until next version to be able to replace this one.
```suggestion
```

Reply:
Haha yeah, sure, my bad, I was not focused on what that method was actually doing.